### DISCUSSION TITLE
Ideas for logo?

### DISCUSSION THREAD

**Discussion by sgbaird** [sparks-baird/matbench-genmetrics#31] [Ideas]
@sparks-baird/xtal2png

Maybe weightlifting emoji/picture 🏋️ with crystal structures or atoms instead of (or on top of) the weights?

@sp8rks want to throw any prompts to your DALLE trial?

e.g.
- "emoji of a a weightlifter lifting atomic structures instead of weights"
- "cartoon of a a weightlifter lifting atomic structures instead of weights"
- "emoji of a a weightlifter lifting atoms instead of weights"

**Comment by kjappelbaum** [comment]
DALLE mini is "interesting" :D 

![Screen Shot 2022-07-31 at 11 46 48](https://user-images.githubusercontent.com/32935233/182020565-8b6180da-33cc-4d66-a036-d5ed6114d444.png)

**Reply by sgbaird** [reply]
DALLE-mini does some interesting things with the human body sometimes 😅

**Comment by sp8rks** [comment]
Not great so far.
